lib/luatest_server.py

```diff
@@ -54,6 +54,8 @@ def execute(self, server):
         if Options().args.pattern:
             for p in Options().args.pattern:
                 command.extend(['--pattern', p])
+        luatest_logfile_path = os.path.join(server.vardir, 'run.log')
```

Comment:
Could you please put this code block before `if`?

Reply:
Fixed.